Authentication/SignUp/TermsAndServices/TermsAndServicesControllerDelegate.swift

```diff
@@ -22,7 +21,7 @@ public protocol TermsAndServicesControllerDelegate {
      
         - Parameter controller: TermsAndServicesController where the event is happening.
      */
-    func didStartLoadingTermsAndServices(controller: TermsAndServicesController)
+    func didStartLoadingTermsAndServices(_ controller: TermsAndServicesController)
```

Comment:
Same name considerations as with login and sign up.
`func didStartLoadingTermsAndServices(in controller: TermsAndServicesController)` or `func didStartLoadingTermsAndServices(controller: TermsAndServicesController)` or `func didStartLoadingTermsAndServices(from controller: TermsAndServicesController)`?